components/board/customlinux/setup_test.go

```diff
@@ -0,0 +1,27 @@
+//go:build linux
+
+// Package customlinux implements a board running linux
+package customlinux
+
+import (
+	"testing"
+
+	"go.viam.com/test"
+)
+
+func TestConfigParse(t *testing.T) {
+	emptyConfigPath := "./data/test_config_empty.json"
+	_, err := parseBoardConfig(emptyConfigPath)
+	test.That(t, err, test.ShouldNotBeNil)
+	test.That(t, err.Error(), test.ShouldContainSubstring, `"name" is required`)
+
+	emptyPWMID := "./data/test_config_emptypwm.json"
+	_, err = parseBoardConfig(emptyPWMID)
+	test.That(t, err, test.ShouldNotBeNil)
+	test.That(t, err.Error(), test.ShouldContainSubstring, `must supply pwm_id for the pwm chip`)
```

Comment:
What's going on with the backticks in these strings? I think double-quotes are the standard string delimiter instead. I think of backticks as being used only in strings that are used for another purpose, like the tags used to get the json package to parse things correctly.